.github/ISSUE_TEMPLATE/feature_template.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: Ruby LSP feature proposal
+description: Propose a new feature for the Ruby LSP
+labels:
+  - enhancement
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Thank you for proposing a new feature! Please answer the following questions to help us the proposal.
+  - type: checkboxes
+    attributes:
+      label: This feature is VS Code specific
+      description: Non VS Code specific features should be proposed to the [Ruby LSP server](https://github.com/Shopify/ruby-lsp/issues/new)
+      options:
+        - label: VS Code specific
+          required: true
+  - type: checkboxes
```

Comment:
I'd maybe drop this one to keep the number of form fields to a reasonable length.

Reply:
Yeah I agree. Perhaps let's not have it yet and see if it's really needed?